views/personal/address.pug

```diff
@@ -25,7 +26,15 @@ block content
                 div #{data.personal.address.line1}
               div #{data.personal.address.city}, #{data.personal.address.province}
               div #{data.personal.address.postalCode}
+  
+  form.cra-form(method='post')
+    +yesNoRadios('confirmAddress', null, 'Is this information correct?', errors)
+      .no-info#noInfo
+        .no-info__wrapper
+          p #{__('If this information doesn’t match your records, you can’t change it here.')}
+          p #{__('You should not use this service to file information you believe to be incorrect.')}
```

Comment:
Looks pretty copy+paste but we're doing a ✨ content audit ✨soon so we can change if needed.

Reply:
It is, but there are no mocks for this content, so it's just generic right now